kaldb/src/main/java/com/slack/kaldb/zipkinApi/ZipkinService.java

```diff
@@ -0,0 +1,52 @@
+package com.slack.kaldb.zipkinApi;
+
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.server.annotation.*;
+import com.slack.kaldb.server.KaldbQueryServiceBase;
+import java.io.IOException;
+import java.util.*;
+
+@SuppressWarnings("OptionalUsedAsFieldOrParameterType")
+public class ZipkinService {
```

Comment:
We should have a comment here that explains what the purpose of the class is, sorta like https://github.com/slackhq/kaldb/blob/master/kaldb/src/main/java/com/slack/kaldb/elasticsearchApi/ElasticsearchApiService.java#L37 or https://github.com/slackhq/kaldb/blob/master/kaldb/src/main/java/com/slack/kaldb/clusterManager/RecoveryTaskAssignmentService.java#L36